src/test/java/ibm/jceplus/junit/base/Record.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright IBM Corp. 2023
+ * Copyright IBM Corp. 2023, 2024
```

Comment:
Do we need to update the copyright if we did not modify the file?

Reply:
I reverted the content of this file and removed the copyright change. While searching and replacing i forgot that we had a few tests that were Junit 5 already so this file was unchanged and does not need a copyright change given that the contents are unchanged.